src/Pages/CategoryPage/CategoryPage.tsx

```diff
@@ -0,0 +1,35 @@
+import { FC, useEffect, useState } from "react";
+import { Content } from "antd/es/layout/layout";
+import { Spin } from "antd";
+import { useParams } from "react-router";
+import { getGoods, getGoodsLoadStatus, goodActions } from "src/store";
+import { useAppDispatch } from "src/hooks/useAppDispatch";
+import { useSelector } from "react-redux";
+import { GoodCategory } from "src/components";
+import { GoodsSearch } from "src/types/general";
+import { useNavigate, Link } from "react-router-dom";
+
+export const CategoryPage: FC = () => {
+    const dispatch = useAppDispatch();
+    const goods = useSelector(getGoods);
+    const { ids } = useParams();
+    const [params] = useState<Partial<GoodsSearch>>({ categoryTypeIds: ids });
```

Comment:
если стейт никогда не изменяется, то это просто переменная, а не стейт
```suggestion
    const params: Partial<GoodsSearch>={ categoryTypeIds: ids };
```